solutions/setup-dev-environment/README.md

```
@@ -16,12 +16,13 @@ sudo chmod +x /usr/local/bin/clusteradm
 
 ## Setup the clusters
 
-Run `./local-up.sh`, you will see two clusters registered on the hub cluster.
+Run `./local-up.sh`, you will see three clusters registered on the hub cluster.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)